tools/MINGW-packages/mingw-w64-pyside/PKGBUILD

```diff
@@ -74,8 +74,6 @@ package() {
     fi
     cd ${srcdir}/build-${MINGW_CHOST}-${pyv}
     make DESTDIR=${pkgdir} install
-    (cd ${pkgdir} ; mv msys*/mingw*/lib/* ${pkgdir}${MINGW_PREFIX}/lib/)
```

Comment:
Removing the ${MINGW_PREFIX} from line 44 removes the need for this and the line below because SITE_PACKAGE is relative to CMAKE_INSTALL_PREFIX.

The rm command is also assuming that msys2 is installed in the root directory of the drive and fails if msys2 was installed elsewhere (e.g. c:\tools\msys64)